### ISSUE TITLE
https://adobe.github.com/ is not a GitHub page any more; documentation should in stead mention https://adobe.github.io/

### ISSUE THREAD

**Issue by jpluimers** [adobe/adobe.github.com#102]
### Expected Behaviour

https://adobe.github.com/ to be alive as a GitHub page.

### Actual Behaviour

https://adobe.github.com/ is not a GitHub page any more

### Reproduce Scenario (including but not limited to)

Browse to https://adobe.github.com/

#### Steps to Reproduce

None (see archived page below under "logs")

#### Platform and Version

Latest Chrome x64 on latest Windows 10 x64.

#### Sample Code that illustrates the problem

None (see archived page below under "logs")

#### Logs taken while reproducing problem

See archived page: https://archive.is/6n7Uf

### Cause

https://adobe.github.com/ used to 302-redirect to https://adobe.github.io until 2019 (see  https://web.archive.org/web/20190607103804/https://adobe.github.com/)

### Proposed change

Change all documentation from pointing to https://adobe.github.com/ or URLs under to the same URL but on the https://adobe.github.io/ subdomain.

At least the toplevel [README.md](https://raw.githubusercontent.com/adobe/adobe.github.com/master/README.md) still points to https://adobe.github.com in two places and should be changed. I have not checked other source files.

**Comment by VictorieeMan** [comment]
This is correct, and the reason for the problem appeared is this change made by GitHub pages:
"Subdomains of github.com are deprecated for GitHub Pages. They will not redirect to github.io after April 15, 2021."

**Comment by AmanNegi** [comment]
Can I work on this issue?

**Comment by Mortecha** [comment]
Would it be worth also changing the name of the repo as well? The issue seems to currently still exist in a different form in the first line of the README as [Adobe GitHub Homepage v2.0](http://adobe.github.com/) seen on the first line leads to a 404 when the link should lead to https://adobe.github.io
